scalding-commons/src/main/scala/com/twitter/scalding/commons/source/DailySources.scala

```diff
@@ -60,6 +60,7 @@ abstract class TimePathedLongThriftSequenceFile[V <: TBase[_, _]: Manifest](f: F
   with WritableSequenceFileScheme
   with Serializable
   with Mappable[(Long, V)]
+  with TypedSink[(Long, V)]
```

Comment:
I think we need to override `override def sinkFields = f` and `override setter[...]` with the subsetter of `TupleSetter.of[(Long, V)]`

And also below.

Might be good to run a quick job (even using local HDFS), to see if it works.